tools/pfe-tools/config.ts

```diff
@@ -35,9 +37,10 @@ export interface PfeConfig {
 const SITE_DEFAULTS: Required<SiteOptions> = {
   description: 'PatternFly Elements: A set of community-created web components based on PatternFly design.',
   favicon: '/brand/logo/svg/pfe-icon-blue.svg',
```

Comment:
I know favicon was mentioned during office hours this morning, but I don't remember was discussed. This is using the old logo. Should we update to `favicon.ico` under `/docs/images/`?